hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -856,22 +859,27 @@ private class Emit(
 
       case x@InitOp(i, args, _, op) =>
         val AggContainer(aggs, sc) = container.get
-        val physicalSignature = aggs(i).lookup(op)
-        args.map(_.pType).zip(physicalSignature.physicalInitOpArgs).foreach { case (l, r) => assert(l == r, s"$l, $r") }
+        val statePTypes = aggs(i).lookup(op).physicalInitOpArgs
         val rvAgg = agg.Extract.getAgg(aggs(i), op)
 
-        val argVars = args.map(a => emit(a, container = container.flatMap(_.nested(i, init = true)))).toArray
+        val argVars = args.zip(statePTypes).map { case (a, t) =>
+          emit(a, container = container.flatMap(_.nested(i, init = false)))
```

Comment:
not to self: this flipped from true to false (but not in other arguers changes. why?